db/migrate/20220324101527_create_likes.rb

```diff
@@ -0,0 +1,7 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] <b>REQUIRED:</b> Please create the corresponding model for likes and implement relationship between `Likes` and other models following the this [ERD Diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png)